src/Setup.php

```diff
@@ -0,0 +1,218 @@
+			*/
+		add_theme_support( 'post-thumbnails' );
+
+		// Add support for Block Styles.
+		add_theme_support( 'wp-block-styles' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
+		// Add alignment option to toolbar options.
+		add_theme_support( 'align-wide' );
+
+		// Add support for custom logos.
+		add_theme_support( 'custom-logo' );
+
+		// Enqueue editor styles.
+		add_editor_style(
+			[
+				'./assets/css/blocks.css',
+			]
+		);
+
+		// Add support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+	}
+
+	/**
+	 * Enqueue scripts and styles for public website.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	public function bcgov_block_theme_enqueue_scripts(): void {
+		$public_assets  = require_once get_template_directory() . '/dist/public.asset.php';
+		$font_assets    = require_once get_template_directory() . '/dist/font.asset.php';
+		$public_version = $public_assets['version'] ?? wp_get_theme()->get( 'Version' );
+		$font_version   = $font_assets['version'] ?? wp_get_theme()->get( 'Version' );
+		wp_enqueue_script(
+			'bcgov-block-theme-public',
+			get_template_directory_uri() . '/dist/public.js',
+			$public_assets['dependencies'] ?? [],
+			$public_version,
+			true
+		);
+		wp_enqueue_style(
+			'bcgov-block-theme-public',
+			get_template_directory_uri() . '/dist/public.css',
+			[],
+			$public_version
+		);
+		wp_enqueue_style(
+			'bcgov-block-theme-font',
+			get_template_directory_uri() . '/dist/font.css',
+			[],
+			$font_version
+		);
+	}
+
+	/**
+	 * Enqueue scripts and styles for admin.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @return void
+	 */
+	public function bcgov_block_theme_enqueue_admin_scripts(): void {
+		$admin_assets  = include_once get_template_directory() . '/dist/admin.asset.php';
+		$admin_version = $admin_assets['version'] ?? wp_get_theme()->get( 'Version' );
+
+		wp_enqueue_script(
+			'bcgov-block-theme-admin',
+			get_template_directory_uri() . '/dist/admin.js',
+			$admin_assets['dependencies'] ?? [],
+			$admin_version,
+			true
+		);
+		wp_enqueue_style(
+			'bcgov-block-theme-admin',
+			get_template_directory_uri() . '/dist/admin.css',
+			[],
+			$admin_version
+		);
+	}
+
+	/**
+	 * Filter the results of get_custom_logo() if no custom logo has been chosen by injecting a default image.
+	 * <-- wp:site-logo --> uses this result internally, and so can be easily used with or without choosing a custom logo.
+	 *
+	 * @param string $html Markup from original get_custom_logo call.
+	 * @since 1.0.0
+	 *
+	 * @return string
+	 */
+	public function bcgov_block_theme_custom_logo( $html ) {
+		// If the site owner has selected a site logo using the theme customizer, use it.
+		if ( has_custom_logo() ) {
+			return $html;
+		}
+		// Otherwise, inject a default logo markup for use in wp-site-logo blocks.
+		$image = '<img src="' . get_stylesheet_directory_uri() . '/dist/images/bc_gov_logo_transparent.ced9da43.png">';
```

Comment:
be careful with this, if this image changes, so will the hash, there might be a more automagic of doing this.